Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Playwright/PlaywrightDriver.cs

```diff
@@ -767,6 +775,27 @@ public List<ElementInfo> GetElementChildren(ElementInfo elementInfo)
             }).Result;
         }
 
+        private string GenerateChildrenXPath(string parentXPath)
+        {
+            string[] parentXPathSegments = parentXPath.Split("/", StringSplitOptions.RemoveEmptyEntries);
+            string elementType = parentXPathSegments[^1];
+
+            int index = elementType.IndexOf("[");
+            if (index != -1)
+            {
+                elementType = elementType.Substring(0, index);
```

Comment:
Try to use .AsSpan() instead of substring

Reply:
handled